requirements_for_test.txt

```diff
@@ -7,5 +7,14 @@ requests-mock==1.9.3
 freezegun==1.1.0
 flake8==3.9.1
 flake8-print==4.0.0
-mypy==0.812
-black==21.5b2
+mypy==0.961
+black==22.3.0
+types-python-dateutil==2.8.17
+types-PyYAML==6.0.8
+types-pytz==2021.3.8
+types-bleach==5.0.2
+types-cachetools==5.0.1
+types-redis==4.2.6
+types-requests==2.27.30
+types-freezegun==1.1.9
```

Comment:
Just curious why you added these `types-` libraries (or perhaps, the real question is why we didn't have them before)?